src/main/java/de/muenchen/isi/domain/model/stammdaten/LookupListsModel.java

```diff
@@ -44,4 +44,6 @@ public class LookupListsModel {
     private LookupListModel artGsNachmittagBetreuung;
 
     private LookupListModel sobonOrientierungswertJahr;
+
+    private LookupListModel sobonOrientierungswertJahrWithoutStandortabfrage;
```

Comment:
Da die meisten Variablen auch eher eine Zusammensetzung deutscher Wörter sind, würde ich das hier ebenfalls so machen.

```suggestion
    private LookupListModel sobonOrientierungswertJahrOhneStandortabfrage;
```

Reply:
In den Coding-Guidelines ist definiert, dass fachliche Benennungen in Deutsch und der Rest in Englisch geschrieben wird.